src/features/unmarkedWsNoticeView/UnmarkedWsNoticeView.tsx

```diff
@@ -106,14 +114,15 @@ const UnmarkedWsNoticeView = ({
 
       {order && (
         <InvoiceCard
+          applicationStatus={noticeDetails.status}
           buttonsRight={
-            <Button variant="secondary" theme="coat" onClick={handleDeleteLease}>
+            <Button variant="secondary" theme="coat" onClick={handleDeleteLease} disabled={isDeleteLeaseLoading}>
```

Comment:
Deleting the application or lease shouldn't be possible once the application has been paid.

This should also have a confirmation modal: https://helsinkisolutionoffice.atlassian.net/browse/VEN-881

Reply:
I'll add the VEN-881 modal to this PR as well. What about the "pending" state? Should the possibility to delete the application/lease be disable for that case as well?

Reply:
Done.